<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>512</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionCameraPerspective"/>
    <addaction name="actionCameraOrthogonal"/>
    <addaction name="actionCameraPushbroom"/>
    <addaction name="actionCameraBilinear"/>
    <addaction name="actionCameraXSlit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionCameraPerspective">
   <property name="text">
    <string>Perspective</string>
   </property>
  </action>
  <action name="actionCameraOrthogonal">
   <property name="text">
    <string>Orthogonal</string>
   </property>
  </action>
  <action name="actionCameraPushbroom">
   <property name="text">
    <string>Pushbroom</string>
   </property>
  </action>
  <action name="actionCameraBilinear">
   <property name="text">
    <string>Bilinear</string>
   </property>
  </action>
  <action name="actionCameraXSlit">
   <property name="text">
    <string>XSlit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
